File: kth_largest_in_bst.java
class Solution
{
    // return the Kth largest element in the given BST rooted at 'root'
    public void helper(Node root, ArrayList<Integer> inorder){
        if(root == null) return;
        helper(root.right, inorder);
        inorder.add(root.data);
        helper(root.left, inorder);
    }
    public int kthLargest(Node root,int K)
    {
        ArrayList<Integer> inorder = new ArrayList<>();
        helper(root, inorder);
        return inorder.get(K-1);
    }
}

File: kth_largest_in_bst.cpp
class Solution
{
public:
    void helper(Node* root, int K, int &count, int &ans){
        if(root == nullptr) return;
        helper(root->right, K, count, ans);
        count++;
        if(K == count){
            ans = root->data;
            return;
        }
        helper(root->left, K, count, ans);
    }
    int kthLargest(Node *root, int K)
    {
        int count = 0;
        int ans;
        helper(root, K, count, ans);
        return ans;
    }
};